packages/app/vite.config.js

```diff
@@ -3,11 +3,11 @@ import path from 'path';
 import { URL, fileURLToPath } from 'url';
 import { defineConfig } from 'vite';
 
-const __dirname = fileURLToPath(new URL('.', import.meta.url));
+const dirname = fileURLToPath(new URL('.', import.meta.url));
```

Comment:
For some reason, the identifier `__dirname` seems to now be statically replaced by a string before the config file is executed, leading to an error. Even though it's replaced by a string, Node still gives me an error when I try to use `__dirname` directly, as doing so is not allowed inside ES modules. So I don't have a choice but to rename the constant.